crates/nu-utils/src/sample_config/default_config.nu

```diff
@@ -825,18 +825,18 @@ $env.config = {
             event: { edit: capitalizechar }
         }
         {
-            name: copy_selection
+            name: copy_selection_system
```

Comment:
May be worth adding a comment that this requires the terminal to support the ctrl-shift bindings passing through (and the `system-clipboard` feature, if it is likely to be disabled on a build).